content/en/docs/reference/glossary/kubelet-eviction.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Node-level eviction
```

Comment:
```suggestion
title: Node-initiated eviction
```
?

Reply:
I think I prefer to highlight the reason, rather than what executes it. So my recommendation is Node-pressure eviction. Although I prefer Node-initiated than Node-level.